### DISCUSSION TITLE
Project still active?

### DISCUSSION THREAD

**Discussion by mikkelking** [nosqlclient/nosqlclient#567] [General]
This looks like it is falling into neglect, the demo site is no longer there, or the docs. Pr's outstanding, although not many.
I can help in a small way if that's helpful. Looks like a useful tool, I'm planning to use it.

**Comment by rsercano** [comment]
Hello,

unfortunately as stated in the README.MD, this was mostly a one man's work and I'm not able to continue working on Nosqlclient due to life conditions. I'll be more than happy to accept your help.
